### ISSUE TITLE
Possibility to add a left separator

### ISSUE THREAD

**Issue by awelormro** [mengelbrecht/lightline-bufferline#130]
Just to ask if there is a way to add a left separator to the bufferline, that's it

**Comment by mengelbrecht** [comment]
Do you mean a custom string on the left of every buffer name?

**Comment by awelormro** [comment]
Exactly, to add a glyph in the left , similar to the airline setting to add a left buffername/tab separator
